src/zero/zero.clj

```diff
@@ -17,7 +17,13 @@
 (def the-github-repos
   "Map Zero source repo names to their URLs"
   (let [repos ["wfl" "dsde-pipelines" "pipeline-config"]
-        git   (partial str "https://github.com/broadinstitute/")]
+        git   (partial str "git@github.com:broadinstitute/")]
+    (zipmap repos (map git repos))))
+
```

Comment:
Factor out the `fn` applied and these are the same.

Reply:
This is true, but not understanding clojure very well I am not quite sure how to combine these two.